ytcomment.py

```diff
@@ -1,21 +1,31 @@
+import re
+from textblob import TextBlob
 from helpers.analysis import calc_sentiment
+
 class YTComment:
     def __init__(self, data):
         self.value = ""
         self.likes = 0
         self.sentiment = 0
-        #
+
         if "snippet" in data:
             if "topLevelComment" in data["snippet"]:
                 if "snippet" in data["snippet"]["topLevelComment"]:
                     real_data = data["snippet"]["topLevelComment"]["snippet"]
                     if "textDisplay" in real_data:
-                        self.value = real_data["textDisplay"]
+                        self.value = spellcheck(real_data["textDisplay"])
                     if "likeCount" in real_data:
                         self.likes = real_data["likeCount"]
         if self.value:
             self.sentiment = calc_sentiment(self.value)
 
+replace_dict = {"&#39;": "'", "&quot;": '"', "\u003cbr /\u003e": " ", "<b>": " ", "</b>": " ", "<i>": " "}
```

Comment:
Put this method inside the `YTComment` class if you are not going to use it in another file.